src/App.tsx

```diff
@@ -13,6 +15,7 @@ export function App() {
           <AppRouter routes={routes} />
         </Layout>
       </BrowserRouterFacade>
+      <CategoryFilter categories={categorySeeker(ProductAPIData)} apiData={ProductAPIData} />
```

Comment:
You don't need to call this component anywhere, that's why we are using storybook and for the props you can use ```defaultProps``` or do it within the ```stories.tsx``` file

Reply:
Thank you about this; 
but I am not clearly getting the reason why I can define new types instead of importing them where I have just already declared them. 
Please help me to understand the defeats of this approach I have used.

Reply:
@ISHIMWE-Aime 

What @SergeInGithub  is saying, is that remove that line 12, and rely on storybook to demo the component.